### DISCUSSION TITLE
Beskrivelse av bruksområder for kompakte og vanlige komponenter

### DISCUSSION THREAD

**Discussion by AudunBerdal** [fremtind/jokul#1816] [Annet]
Etterlyser dokumentasjon i Jøkul på forskjellen mellom kompakte og vanlige versjoner av komponentene og når disse skal brukes / ikke brukes. Hvis det er slik at kompakt-versjoner kun skal brukes i interne informasjonstunge ekspertsystemer, bør kanskje dette beskrives eksplisitt. 

Jeg har sett flere Figma-skisser (deriblant mine egne) der kompakt brukes i mobilskisser for kundeløsninger, så dette er det nok flere som tar feil av. Vi har mange ulike typer designere som ikke nødvendigvis kjenner minstemål for trykkflater på mobil. 

**Forslag:** 

- Legge inn et avsnitt om dette under Kom i gang > Mobil. Her står det en linje om minste trykkflate på 44x44 px, men ikke at kompakt-versjonene har mindre flate en dette. 

- Legge inn noe om anbefalte mål for trykkflate i oppslagsguiden vår for universell utforming.

**Comment by Slaine2** [comment]
Jeg tror vi også kan justere teksten til de nye komponentene (Varianter) under Kompakt. Dette gjør det lettere å forstå hvor du skal bruke dem.

https://www.figma.com/file/y9JtTf3whfTCHCdYSUXVQ5/%F0%9F%9B%A0-J%C3%B8kul-komponenter-Pull-request?node-id=84%3A2399
